polaris-core/src/main/java/org/apache/polaris/core/persistence/LocalPolarisMetaStoreManagerFactory.java

```diff
@@ -178,20 +178,22 @@ public synchronized Supplier<TransactionalPersistence> getOrCreateSessionSupplie
 
   @Override
   public synchronized StorageCredentialCache getOrCreateStorageCredentialCache(
-      RealmContext realmContext) {
+      RealmContext realmContext, PolarisCallContext polarisCallContext) {
```

Comment:
You can get from PolarisCallContext to RealmContext, so I don't see a point in passing in two arguments

Reply:
> You can get from PolarisCallContext to RealmContext

Is that true? I think you can get from CallContext to RealmContext, but not from PolarisCallContext. Personally I would like to unify these in the future but I want to keep this PR tightly scoped.